drivers/staging/vc04_services/interface/vchiq_arm/vchiq_arm.c

```diff
@@ -64,10 +64,10 @@
 #define VCHIQ_MINOR 0
 
 /* Some per-instance constants */
-#define MAX_COMPLETIONS 16
+#define MAX_COMPLETIONS 128
```

Comment:
The commit messages doesn't explain why it's increased. So this should be a separate patch.